### ISSUE TITLE
Random State while splitting

### ISSUE THREAD

**Issue by kinoute** [amaiya/ktrain#34]
Hey there,

I think the user should be able to set a random state, when for example, loading texts from folder for pre-processing. By default, if no test folder is given, 10% of the training set is taken as validation set. But this is done differently every time, i.e. the data are taken randomly.

This is a problem when you want to compare different models because the data are not splitted the same way. I see you use `load_files` and `train_test_split` both from sklearn. Both accept a `random_state` argument according to their documentation.

It would be great if we could pass it too. I can make a PR if you want.

**Comment by amaiya** [comment]
Agreed - this should be added.   (Temporary workaround is to load the data once and then compare different models on the same split.)

**Comment by kinoute** [comment]
Sure, that could work. Unfortunately, if you want to compare a BERT model that needs a different pre-processing, or a bi-GRU which needs a n-gram set to 1, you won't be able to.

**Comment by amaiya** [comment]
Yes, you're right.  I guess the only temporary workaround, then, would be to pre-split your data and read-in both training and validation sets using `texts_from_folder`.  But, a `random_state` argument will be added to all data-loading functions, so this won't be necessary.  Good suggestion.

**Comment by amaiya** [comment]
*ktrain* v0.7.1 includes a `random_state` parameter to `texts_from_folder` and all other data-loading functions.  If setting `random_state` to a fixed integer (e.g., `random_state=42`), the same train-validation split can be reproduced:
```
(x_train, y_train), (x_test, y_test), preproc = text.texts_from_folder('data/corpus', 
                                                                       maxlen=100, 
                                                                       max_features=30000,
                                                                       train_test_names=['train'],
                                                                       val_pct=0.1,
                                                                       random_state=42)
```
